crates/lib/src/stdlib/filters/string/truncate.rs

```diff
@@ -72,13 +72,15 @@ impl Filter for TruncateFilter {
         let args = self.args.evaluate(runtime)?;
 
         let lenght = args.lenght.unwrap_or(50) as usize;
-
         let truncate_string = args.ellipsis.unwrap_or_else(|| "...".into());
-
-        let l = cmp::max(lenght - truncate_string.len(), 0);
+        let diff = if lenght >= truncate_string.len() {
```

Comment:
Oh `lenght`.  This is why I want to get `typos` done so I can run it in CI :)

Reply:
Oh, didn't see it was wrong… #french :p I'll fix it :)